RNTester/js/OutOfTreeTestPlatform/Libraries/Settings/Settings.dummy.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ * @todo Implement
+ */
+
+'use strict';
+
+// Completely empty shim for the sake of bundling.
+const Settings = {};
```

Comment:
if `require('UnimplementedView');` can't work here, let's inline the assignment as elsewhere.